src/api/README.md

```diff
@@ -46,7 +46,7 @@ This function takes user-provided data and uses it to create a new item in the F
 
 #### Note
 
-**`daysUntilNextPurchase` is not added to the item directly**. It is used alomngside the `getFutureDate` utility function to create a new _JavaScript Date_ that represents when we think the user will buy the item again.
+**`daysUntilNextPurchase` is not added to the item directly**. It is used alongside the `getFutureDate` utility function to create a new _JavaScript Date_ that represents when we think the user will buy the item again.
```

Comment:
nice catch :) I live and die by spell check in my ide for this reason. I shipped one too many typos to production, I swore I'd never do it again (so far so good 🚀 )

Reply:
yeah I learned long ago just how bad I can be at spelling 😅 so I looked for a [spell checker extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) in VS to support me when I fail 😂